{{cookiecutter.project_slug}}/clients/web/vue3/src/views/Login.vue

```diff
@@ -20,9 +20,6 @@
         />
         <div>
           <div class="flex items-center justify-between">
-            <label for="password" class="block text-sm font-medium leading-6 text-primary"
```

Comment:
This was unnecessary since it is part of the component and without it the component adds a undefined

Reply:
Fair but it was in here so that Password and Forgot Password could sit side by side -- I think it's fine like this but forgot password should sit on the rhs

Reply:
Ah I see the only reason I moved it was because of the weird `undefined-label`
Would it be better located under create and account? or the RHS of that

Reply:
Ahh also that means the label for wasnt working!

Reply:
Done I made it into a slot!